src/ResourceManager/Sql/Commands.Sql/help/Stop-AzureRmSqlElasticPoolActivity.md

```diff
@@ -0,0 +1,171 @@
+﻿---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version:
+schema: 2.0.0
+---
+
+# Stop-AzureRmSqlElasticPoolActivity
+
+## SYNOPSIS
+Cancels the asynchronous update operation on an elastic pool.
+
+## SYNTAX
+
+```
+Stop-AzureRmSqlElasticPoolActivity [-ServerName] <String> [-ElasticPoolName] <String> [-OperationId <Guid>]
+ [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Stop-AzureRmSqlElasticPoolActivity** cmdlet cancels the asynchronous update operation on an elastic pool.
+
+## EXAMPLES
+
+### Example 1: Cancel the asynchronous update operation on an elastic pool
+```
+PS C:\> Stop-AzureRmSqlElasticPoolActivity -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -ElasticPoolName "ElasticPool01" -OperationId af97005d-9243-4f8a-844e-402d1cc855f5
+```
```

Comment:
@payiAzure please remove the three backticks on this line

Reply:
@cormacpayne removed in the new commit.